tools/triage/ticket_search/cli/app.py

```diff
@@ -0,0 +1,93 @@
+"""The module for the main cli app"""
+# pylint: disable=too-few-public-methods
+import argparse
+import logging
+import sys
+
+from command import Command
+from fetch_by_id import FetchById
+from generate_json_report import GenerateJSONReport
+from search_by_path_and_content import SearchByPathAndContent
+from search_for_tickets import SearchForTickets
+from workflow_data import WorkflowData
+
+
+class App:
+    """General launcher app for command line utilities"""
+
+    def __init__(self, arguments: list[str]):
+        self.args = arguments
+        self.logger = logging.getLogger(__name__)
+
+    def run(self, logging_verbosity):
+        """App entry point"""
+
+        self.logger.debug("Starting...")
```

Comment:
```suggestion
        self.logger.info("Starting...")
```

Reply:
so we'd start with some input (currently it doesn't output anything which looks to be stuck)

Reply:
actually as we use stdio to output the actual json document and we want to be able to pipe this somewhere, the optimal behaviour is to only show this in debug mode

Reply:
yeah, that's why I thought to have the logs in stderr and the output in stdout
this way piping commands works out of the box and yet you can see the errors if there are any